src/streetscapes/project.py

```diff
@@ -0,0 +1,149 @@
+from pathlib import Path
+
+import ibis
+from shapely.geometry import box
+
+from streetscapes import config
+
+
+class Project:
+    """Minimal project managing a DuckDB/Ibis connection."""
+
+    def __init__(self, name: str = "streetscapes"):
+        # TODO: also read name from config? But keep option to overwrite?
+        self.name = name
+        self.data_home = Path(config.get("data_home"))
+
+        database_path = self.data_home / "projects" / f"{name}.duckdb"
+        database_path.parent.mkdir(parents=True, exist_ok=True)
+        self.con = ibis.duckdb.connect(database_path)
+        self.con.raw_sql("INSTALL spatial; LOAD spatial;")
+
+    def image_dir(self, source: str | None = None):
+        if source is None:
+            return self.data_home / "images"
+        return self.data_home / "images" / source
+
+    def get_table(self, name: str):
+        """Return an ibis table reference."""
+        return self.con.table(name)
+
+    def ingest_mapillary(self, df, table_name="mapillary"):
+        """Ingest a DataFrame of Mapillary metadata."""
+        self.con.con.register("metadata_tile", df)
+        if table_name not in self.con.list_tables():
+            self.con.raw_sql(f"""
+                CREATE TABLE {table_name} AS
+                SELECT
+                    * EXCLUDE (geometry),
+                    ST_GeomFromText(geometry) AS geometry,
+                FROM metadata_tile;
+                ALTER TABLE {table_name} ADD PRIMARY KEY (id);
+            """)
+        else:
+            # TODO: consider configurable duplicate behaviour (REPLACE or IGNORE)
+            self.con.raw_sql(f"""
+                INSERT OR REPLACE INTO {table_name}
+                SELECT
+                    * EXCLUDE (geometry),
+                    ST_GeomFromText(geometry) AS geometry,
+                FROM metadata_tile
+            """)
+
+    def filter_bbox(self, table_name, bbox):
```

Comment:
(Also import `Bbox` from `fetch_metadata.py` if that's the right type)

```suggestion
    def filter_bbox(self, table_name: str, bbox: Bbox):
```